openedx/core/lib/docs/decisions/0001-logging-and-monitoring-ignored-errors.rst

```diff
@@ -14,7 +14,7 @@ We had two recent Production issues that took longer than necessary to diagnose
 Decision
 ________
 
-Add a capability for logging and monitoring ignored errors and expected errors. The feature will be configurable, the monitoring will be available by default, and the logging will be opt-in as needed.
+Add a capability for logging and monitoring ignored errors. The feature will be configurable, the monitoring will be available by default, and the logging will be opt-in as needed.
```

Comment:
@Yagnesh1998: Good find! This is an ADR (Architectural Decision Record). However, because ADRs should be mostly immutable (i.e. unchanged), we'll introduce a new ADR instead and leave this one mostly unchanged. I'll help with that, and will probably create a new branch/PR to show you what I mean.

Reply:
@Yagnesh1998: can you bring in these changes? https://github.com/openedx/edx-platform/pull/33370

Reply:
@robrap Sure, I'll do that.Thanks.